### ISSUE TITLE
Any similar script for Region bypass?

### ISSUE THREAD

**Issue by Jaladhjin** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#162]
### What's the problem this feature will solve?

Thanks very much for this do you know of any similar script that will do a Region bypass?

### Describe the solution you'd like

Thanks very much for this do you know of any similar script that will do a Region bypass?

**Comment by drunkwinter** [comment]
No, I don't. But I guess you are talking about a proxy/VPN here? There are probably plenty of proxies/VPNs available many of which are paid.

You have another option, which is renting your own server at some location and then changing the proxy URL setting in our bypass to point to that server. The downside will be that it's just one location compared to a VPN and I think that this is probably too much hassle for you in which case a VPN is the best option.

Fun fact is we have a proxy built-in to the bypass, but just one server. And as far as I remember located in the US.
